templates/mainapp/layout.html

```diff
@@ -119,8 +119,18 @@ <h4 class="modal-title" id="booktitle">Basic Modal</h4>
                             <strong>Authors:</strong>
                             <p id="bookauthors">
                             </p>
+                            <!-- Rating Block -->
 
-                            <div id="bookrating"></div>
+                            <div class="ratings" id="rate">
+                                <form class="rate-form" action="" method="POST">
+                                    {% csrf_token %}
+                                    <button type="submit" class="fa fa-star star-btn" id="first"></button>
```

Comment:
Hi, this is a nice implementation of yours for rating, though I would recommend to reuse the implementation in [layout.html](https://github.com/Praful932/Kitabe/blob/master/templates/mainapp/layout.html#L271) :)